bin/convert-env.sh

```diff
@@ -9,6 +9,49 @@
 
 OSNAME=$(uname)
 
+convert_v2_to_v1() {
+  while read old_name new_name; do
+    old_val=$(eval echo "\$${old_name}")
+    new_val=$(eval echo "\$${new_name}")
+    if [ -z "${old_val}" -a -n "${new_val}" ]; then
+      export "${old_name}=${new_val}"
+    fi
+  done <<EOF
+CATALOG_PORT ZWE_components_api_catalog_port 
+DISCOVERY_PORT ZWE_components_discovery_port
+GATEWAY_HOST ZWE_haInstance_hostname
+GATEWAY_PORT ZWE_components_gateway_port
+KEY_ALIAS ZWE_zowe_certificate_keystore_alias
+KEYSTORE ZWE_zowe_certificate_keystore_file
+KEYSTORE_CERTIFICATE ZWE_zowe_certificate_pem_certificate
+KEYSTORE_CERTIFICATE_AUTHORITY ZWE_zowe_certificate_pem_certificateAuthority
+KEYSTORE_CERTIFICATE_AUTHORITY ZWE_zowe_certificate_truststore_certificateAuthorities
```

Comment:
After discussed with Aditya and Andrej, we think `KEYSTORE_CERTIFICATE_AUTHORITY ZWE_zowe_certificate_pem_certificateAuthorities` should work better with less confusing.

If i remember correctly why v1 has 2 CA entries is trying to align with 2 env vars `EXTERNAL_ROOT_CA` and `EXTERNAL_CERTIFICATE_AUTHORITIES`. We can take the chance to converge those 2 PEM format CA entries.